packages/eslint-plugin-internal/src/rules/react-typescript-prettier.js

```diff
@@ -13,4 +13,7 @@ module.exports = {
 
   // Enforce a defaultProps definition for every prop that is not a required prop
   'react/require-default-props': 'off',
+
+  // Enforce that block statements are wrapped in curly braces
+  curly: [2, 'all'],
```

Comment:
I'd suggest using string-based rule severity values instead of their numeric counterparts for better readability.

```suggestion
  curly: ['error', 'all'],
```